Frontend/ui-customer/src/Components/DriverInfoCard/View.purs

```diff
@@ -1281,3 +1236,6 @@ getVehicleImage variant vehicleDetail city = do
       mkVehicleImage cityCode = 
         if cityCode == "std:040" then "ic_auto_rickshaw_black_yellow"
           else "ic_auto_rickshaw"
+
+getAnimationDelay :: LazyCheck -> Int
```

Comment:
Why lazy check is required for constant, what value will it give if no lazy check?

Reply:
Animation is not working without it